src/test/spec/unified_runner/test_file.rs

```diff
@@ -33,6 +37,9 @@ pub struct TestFile {
     pub create_entities: Option<Vec<TestFileEntity>>,
     pub initial_data: Option<Vec<CollectionData>>,
     pub tests: Vec<TestCase>,
+    // TODO: don't read this field
```

Comment:
Is this TODO copied from the unified spec test runner, or did you add this?

Reply:
I added this. I didn't find a way to ignore a field in serde, so I'll remove this comment and leave the field as-is.

Reply:
You can have serde ignore a field with [`#[serde(skip)]`](https://serde.rs/field-attrs.html#skip).

Reply:
Gave this a shot, but I believe that the `deny_unknown_fields` option prevents this from working. Using `#[serde(skip)]`, even in combination with `rename`, causes a panic about `_yamlAnchors` being an unknown field:
> unknown field `_yamlAnchors`, expected one of `description`, `schemaVersion`, `runOnRequirements`, `allowMultipleMongoses`, `createEntities`, `initialData`, `tests`

I've removed the todo comment, but it looks like we'll have to consume the field from the test files and then ignore it.